include/flatbuffers/flatbuffers.h

```diff
@@ -639,18 +639,22 @@ class vector_downward {
     allocated_bytes = reserved_;
     offset = static_cast<size_t>(cur_ - buf_);
 
+    // release_raw only relinquishes the buffer ownership.
+    // Does not deallocate or reset the allocator. Destructor will do that.
```

Comment:
Ok, that seems indeed nicer than what we had before.. change the minimum possible.